test/java/seedu/addressbook/util/TestUtil.java

```diff
@@ -75,4 +77,18 @@ public static void assertTextFilesEqual(Path path1, Path path2) throws IOExcepti
         List<String> list2 = Files.readAllLines(path2, Charset.defaultCharset());
         assertEquals(String.join("\n", list1), String.join("\n", list2));
     }
+
+    /**
+     * Executes the command, and asserts the result message is as expected.
+     */
+    public static void assertCommandResult(Command command, String expectedMessage, AddressBook addressBook, 
+                                                                             AddressBook expectedAddressBook) {
```

Comment:
Similarly, can reorder parameters so that input comes first and expected values come next.